packages/core/src/codewhispererChat/controllers/chat/messenger/messenger.ts

```diff
@@ -88,6 +90,24 @@ export class Messenger {
             )
         )
     }
+
+    public async countTotalNumberOfCodeBlocks(message: string): Promise<number> {
```

Comment:
This makes much more sense than counting triple backticks. However, for efficiency, we still need to get it directly from the response. But for a client implementation good work👍🏻